gmusicapi/clients/musicmanager.py

```diff
@@ -1,17 +1,19 @@
 from __future__ import print_function, division, absolute_import, unicode_literals
-from future import standard_library
-standard_library.install_aliases()
+from future.utils import PY3
 from past.builtins import basestring
 from builtins import *  # noqa
+
 import os
 from socket import gethostname
 import time
-import urllib.request
-import urllib.parse
-import urllib.error
 from uuid import getnode as getmac
 import webbrowser
 
+if PY3:
```

Comment:
> My understanding was that it was preferable to check if PY2. Is there some nuance I didn't understand?

I dunno if one way is better than the other. If you were being super explicit, you'd check for both (e.g. requests does this in its compat module). But that'd be pretty pointless. I can't really think of a good reason to check for one over the other.

> Could this just use six.moves.urllib.parse?

Why bother using another compat library when it can be a simple version check? If anything, I was thinking about removing the PY3 import and just doing a ``PY3/is_py3 == sys.version_info[0] == 3`` since this is all that's being done on future's end anyway. I wanted to keep the simple syntax for any novice people reading it to quickly understand. This will all be removed anyway once we get to the point where we can kill Python 2 support anyway : P